NebulaWorld/Player/DroneManager.cs

```diff
@@ -13,6 +13,14 @@ public static class DroneManager
         public static List<int> PendingBuildRequests = new List<int>();
         public static Dictionary<ushort, Vector3> CachedPositions = new Dictionary<ushort, Vector3>();
 
+        public static void Initialize()
+        {
+            DronePriorities = new int[255];
```

Comment:
They're already initialized in the constructor?

Reply:
Why is an explicit initialize call needed here?

Reply:
It's probably to reset all the collections each time a player host or join a new game without restarting the game.

Reply:
I would say that removing the initializers in the constructor (the ones above) is the best way to figure out if something uses the service before initialization. Or have everything done in a session object that can be referred to for all the current session managers.

Reply:
Since the fields above are static, they are persistent when user disconnect and connect again. This servers as "Cleaning", so every time user connects to the game, the arrays will be fresh and clean.

Reply:
I put it on my list to look at making a `Session` object that's refreshed on every connection.